sdk/typescript/introspector/test/scan.spec.ts

```diff
@@ -37,6 +37,7 @@ describe("scan static TypeScript", function () {
                   description: "",
                   optional: false,
                   defaultValue: undefined,
+                  isVariadic: false,
```

Comment:
> 1. Test individual parts in separate test cases
Instead of asserting the whole structure, assert individual fields that you care about in tests. This is what we do mostly in integration tests, for example, with module introspection and JSON querying. Then if you add a new field, you don't have to update all the test cases, just the new test that queries for it specifically.

I agree with you but the thing is that it will take as much time to specify specific field inside the `assert`.
And testing everything was actually useful because I found issues in other cases in #6537

I'm down to work on the golden part in a follow up though, I'll need more details to understand this